napari/_qt/qt_main_window.py

```diff
@@ -496,7 +496,8 @@ def close(self, quit_app=False, confirm_need=False):
         if (
             not confirm_need_local
             or not get_settings().application.confirm_close_window
-            or ConfirmCloseDialog(self, quit_app).exec_() == QDialog.Accepted
+            or ConfirmCloseDialog(self, quit_app).exec_()
+            == QDialog.DialogCode.Accepted
```

Comment:
It is not related to the main topic of PR, but I put it here to not create single line PR. 

Just proper Enum usage.